zschema/compounds.py

```diff
@@ -11,12 +11,13 @@ def _is_valid_object(name, object_):
 
 class ListOf(Keyable):
 
-    def __init__(self, object_, required=None, max_items=None, doc=None, category=None):
+    def __init__(self, object_, required=None, max_items=None, doc=None, category=None, validator=None):
```

Comment:
This doesn't look like it's based off of master, which now uses `_NO_ARG`.

Reply:
> This doesn't look like it's based off of master, which now uses `_NO_ARG`

This branch was forked off of master before https://github.com/zmap/zschema/pull/35 was merged. Should this PR pull in `master` before merge to use `from keys import _NO_ARG`, or can it merge as is?

Reply:
I mean.... it conflicts with master.... so no?

Reply:
https://github.com/zmap/zschema/pull/42/commits/09fd51679ec0e33e24b45b0c35d552d61835cf53 merges `master` in